src/pages/Tailwind/Pool/ExpandablePoolRow.tsx

```diff
@@ -54,12 +55,23 @@ interface ExpandablePoolRowProps {
 
 const ExpandablePoolRow = ({ poolAddress, pid, isExpanded, onClick, isActivePool = true }: ExpandablePoolRowProps) => {
   const [pool, setPool] = useState<PoolData | undefined>(undefined)
+  const [reloader, setReloader] = useState(0); //eslint-disable-line
```

Comment:
```suggestion
  const [reloader, setReloader] = useState(0);
```

The reason why you are getting a warning here is probably because `reloader` is not used. It should be added to the useEffect dependencies array on line 135 as well.